cmd/limactl/copy.go

```diff
@@ -117,9 +128,12 @@ func copyAction(cmd *cobra.Command, args []string) error {
 
 	sshCmd := exec.Command(arg0, append(sshArgs, scpArgs...)...)
 	sshCmd.Stdin = cmd.InOrStdin()
-	sshCmd.Stdout = cmd.OutOrStdout()
-	sshCmd.Stderr = cmd.ErrOrStderr()
```

Comment:
These lines can be kept.

`scp` should be invoked with `-q` unless `--verbose` is set.

Reply:
The` -q `flag alone doesn't seem to quiet the logs as expected. I’ve attached a screenshot to illustrate the issue. The first image shows the args or flags and the next shows the output from the action. @AkihiroSuda 


<img width="861" alt="Screenshot 2024-12-24 at 17 25 35" src="https://github.com/user-attachments/assets/8f0b5c44-dd21-4bc6-ae06-a8451c544c80" />
<img width="970" alt="Screenshot 2024-12-24 at 17 25 53" src="https://github.com/user-attachments/assets/e8d16ec5-5733-48ce-a2b7-279c1ec44b18" />

Reply:
Permission errors should not be silenced.

Reply:
Okay. Got that!

Reply:
@AkihiroSuda How do I update the documentation to reflect this new change? I have checked through the repository but couldn't find any pointer.

Reply:
No need to update the docs this case IMHO

Reply:
https://lima-vm.io/docs/reference/limactl_copy/ will be automatically updated after merging this PR